src/ApiInterface/include/ApiInterface/Orbit.h

```diff
@@ -312,6 +328,8 @@ struct orbit_api_v1 {
   uint32_t enabled;
   uint32_t initialized;
   void (*start)(const char* name, orbit_api_color color, uint64_t group_id);
+  void (*start_with_explicit_caller)(const char* name, orbit_api_color color, uint64_t group_id,
+                                     uint64_t caller_address);
```

Comment:
It's unfortunate that we have to duplicate the function. Could we always take in the new parameter instead? We could either:
1. define a GET_PC() macro to get the current PC which would be use for the non-scope case
2. pass in 0 for the non-scope case and use GET_CALLER_PC if the passed in address is 0

WDYT? I think this would avoid duplication, especially if we start getting the caller pc for other functions which seems to me like we should do (start_async, stop_async, trackX...)

Reply:
Thats a good idea. I implemented suggestion 2. PTAL.